src/app/api/__tests__/ApiTest.ts

```diff
@@ -1,52 +1,106 @@
-import {default as API} from "../index";
-import { DaysOfWeek } from "../Api";
+import {default as API } from '../index';
+import { DaysOfWeek } from '../Api';
  
-describe('API Client',()=>{
+describe('API Client',() => {
 
-    beforeEach(()=>{
-      console.log("before api test")  
+    beforeEach(() => {
+      // tslint:disable-next-line:no-console
+      console.log('before api test');  
     });
 
-    afterEach(()=>{
-        console.log('Cleanup')
+    afterEach(() => {
+        // tslint:disable-next-line:no-console
+        console.log('Cleanup');
     });
 
-
-    it('Should return sheriffs',async ()=>{
+    it('Should return sheriffs',async () => {
         let sheriffs = await API.getSheriffs();
         expect(sheriffs).toBeDefined();
-        //expect(Array.isArray(sheriffs)).toBeTruthy();
-        // let s = sheriffs[0];
-        // expect(s).toMatchObject(Data[0]);
-    })
+    });
 
-})
+});
 
 describe('DaysOfWeek.getDisplayValues', () => {
     const util = DaysOfWeek.getDisplayValues;
     
     it('Should return each day if weekdays and Sat or Sun is selected', () => {
-        expect(util(DaysOfWeek.Weekdays | DaysOfWeek.Sat)).toEqual(["Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]);
-        expect(util(DaysOfWeek.Weekdays | DaysOfWeek.Sun)).toEqual(["Mon", "Tue", "Wed", "Thu", "Fri", "Sun"]);
-    }) 
+        // tslint:disable-next-line:no-bitwise
+        expect(util(DaysOfWeek.Weekdays | DaysOfWeek.Sat)).toEqual(['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat']);
```

Comment:
You can also disable a particular lint rule for a block as well as seen here:

https://github.com/bcgov/jag-shuber-frontend/blob/488b6a31da9bd62edf3aa1251842ccb71979b4a6/src/app/api/Api.ts#L15-L28

Reply:
note, scroll to the bottom of code block ⬆️